auth/auth/auth.py

```diff
@@ -136,6 +137,74 @@ def get_flow(redirect_uri, state=None):
     })
 
 
+@routes.get('/users')
+@web_authenticated_developers_only()
+async def get_users(request, userdata):
+    dbpool = request.app['dbpool']
+    async with dbpool.acquire() as conn:
+        async with conn.cursor() as cursor:
+            await cursor.execute('SELECT * FROM users;')
+            users = await cursor.fetchall()
+    page_context = {
+        'users': users
+    }
+    return await render_template('auth', request, userdata, 'users.html', page_context)
+
+
+@routes.post('/users')
+@check_csrf_token
+@web_authenticated_developers_only()
+async def post_create_user(request, userdata):  # pylint: disable=unused-argument
+    session = await aiohttp_session.get_session(request)
+    dbpool = request.app['dbpool']
+    post = await request.post()
+    username = post['username']
+    email = post['email']
+    is_developer = post.get('is_developer') == '1'
+
+    async with dbpool.acquire() as conn:
+        async with conn.cursor() as cursor:
+            await cursor.execute(
+                '''
+INSERT INTO users (state, username, email, is_developer)
+VALUES (%s, %s, %s, %s);
+''',
+                ('creating', username, email, is_developer))
+            user_id = cursor.lastrowid
+
+    set_message(session, f'Created user {user_id} {username}.', 'info')
```

Comment:
Above the state is “creating”, so I think the message should reflect that?

Reply:
Meh.  I'm happy with how it is.  From the perspective of the user, it got created.